lib/kino/livebook_output.ex

```diff
@@ -0,0 +1,58 @@
+defmodule Kino.LivebookOutput do
```

Comment:
Not sure I would call it LivebookOutput. One of the paths we might explore in the future is to render the same output in different media. So, in order to not paint ourselves into a corner, I would call this `Kino.Output`.

Reply:
Yeah, but this is specific to `to_livebook` and the functions return Livebook-compatible outputs. If we were to output something different then we would add `to_text` function to the `Kino.Render` protocol, but that wouldn't use this module at all, right?

Reply:
That's the thing, I don't know yet... I can see our render output be generic enough to render in multiple places. For example, I can see a table() working on both livebook and also other static scenarios. So I would go with a generic name just in case.

If you want to remain consistent, you can also rename the `to_livebook` function to something generic too, such as `to_output`. And then we can add specializations later on (such as `to_livebook`, `to_console`, etc). WDYT?

Reply:
Actually, now that I wrote it, I like `Kino.Output` and `Kino.Render.to_output` a lot. WDYT?

Reply:
It just kinda covers the fact that the output is Livebook-compatible, but sounds alright to me. One thing to keep in mind is that Livebook will depend on the function name so changing it later may be awkward.

I guess your point is that if we have `to_console` later on, it could still call `to_output` and then render that output on its own, rather than working with the plain value?

Reply:
I was thinking `to_output` would always work and be the general fallback, but we would allow specializations on `to_livebook` and `to_console` or similar?

Reply:
So for me `Kino.Output` should return formats compatible with Livebook, which means `to_output` and `to_livebook` are essentially the same.

If we have a conceptually generic `to_output` then the difference would likely lie in `Kino.render` rather than the protocol implementation right?

Reply:
Sure. I think Kino.Output with Kino.Render.to_livebook works fine too!

Reply:
Actually, if we assume that every `Kino.Output.t()` is a valid `Livebook.Notebook.Cell.output()`, then this looks quite nice:

```elixir
defmodule Livebook.Evaluator.DefaultFormatter do
  ...

  defp to_output(value) do
    # Kino is a "client side" extension for Livebook that may be
    # installed into the runtime node. If it is installed we use
    # its more precies output rendering rules.
    if Code.ensure_loaded?(Kino.Render) do
      try do
        Kino.Render.to_output(value)
      catch
  ...
```

Reply:
It is your call. Both `to_output`/`to_livebook` are fine by me! Go with `to_livebook` if you want to be conservative.